picli/command/sast.py

```diff
@@ -15,22 +15,16 @@ def __init__(self, base_config, debug):
     def execute(self):

```

Comment:
Can we merge these two executes into base.Base?

How you have it, the sast_pipe_config.run_config, sast_module, sast_analyzer and Log messages etc. all are almost identical and could likely be parametrized. Maybe set some class properties and use those values in execute()